var/spack/repos/builtin/packages/gcc/package.py

```diff
@@ -424,7 +424,7 @@ def write_rpath_specs(self):
                               self.prefix.lib, self.prefix.lib64))
         set_install_permissions(specs_file)
 
-    def setup_environment(self, spack_env, run_env):
+    def setup_run_environment(self, run_env):
```

Comment:
I would rename `run_env` to `env` in this method to be consistent with other uses.